src/porepy/models/poromechanics.py

```diff
@@ -137,24 +137,55 @@ class SolutionStrategyPoromechanics(
     :class:`~porepy.models.constitutive_laws.DarcysLaw`.
 
     """
-
     mdg: pp.MixedDimensionalGrid
     """Mixed dimensional grid."""
 
+    biot_tensor: Callable[[list[pp.Grid]], pp.ad.Operator]
+    """Method that defines the Biot tensor. Normally provided by a mixin instance of
+    :class:`~porepy.models.constitutive_laws.BiotCoefficient`.
+    """
+
+    def __init__(self, params: Optional[dict] = None) -> None:
+        """Initialize the solution strategy.
+
+        Parameters:
+            params: Dictionary of parameters.
+
+        """
+        # Initialize the solution strategy for the fluid mass balance subproblem.
+        mass.SolutionStrategySinglePhaseFlow.__init__(self, params=params)
+        # Initialize the solution strategy for the momentum balance subproblem.
+        momentum.SolutionStrategyMomentumBalance.__init__(self, params=params)
+
+        self.biot_coefficient_key = "biot_alpha"
```

Comment:
Can't this be a class attribute? Possibly not, but if so, I'm not sure I like this solution of having to explicitly call the other inits.

Reply:
I think it can be deleted, old code. Thanks for spotting.